oximeter/db/src/oxql/query/mod.rs

```diff
@@ -362,6 +366,27 @@ impl Query {
     pub(crate) fn parsed_query(&self) -> &QueryNode {
         &self.parsed
     }
+
+    /// Insert filters after the `get`, or in the case of subqueries, recurse
+    /// down the tree and insert them after each get.
+    pub(crate) fn add_filters(&self, filters: Vec<Filter>) -> Self {
+        Self {
+            parsed: self.parsed.add_filters(filters),
+            end_time: self.end_time,
+        }
+    }
+}
+
+// TODO: move this to a more appropriate spot
+/// Just a helper for creating a UUID filter node concisely
+pub fn uuid_eq_filter(key: String, id: Uuid) -> Filter {
+    let simple_filter = SimpleFilter {
+        ident: Ident(key),
+        cmp: Comparison::Eq,
+        value: Literal::Uuid(id),
+    };
+    let filter_expr = FilterExpr::Simple(simple_filter);
+    Filter { negated: false, expr: filter_expr }
```

Comment:
This is fine here, I guess, but it's a pretty random helper. I feel like it needs a more specific home.

Reply:
I agree. I think we probably want to only have functions for inserting UUIDs specifically for the scoping we care about, silo or project. I might do that with the enum I mentioned above, or something similar.

Reply:
With the change to `insert_authz_filters` taking the authz scope directly, to me this is now fine here because it's a private helper in this file only.